sdk/keyvault/keyvault-admin/test/public/backupClient.abort.spec.ts

```diff
@@ -61,13 +59,12 @@ describe("Aborting KeyVaultBackupClient's requests", () => {
 
   it("can abort beginSelectiveRestore", async function() {
     const backupURI = `${blobStorageUri}/${generateFakeUUID()}`;
-    const folderName = getFolderName(backupURI);
 
     const controller = new AbortController();
     controller.abort();
 
     await assertThrowsAbortError(async () => {
-      await client.beginSelectiveRestore(blobStorageUri, blobSasToken, folderName, "Key Name", {
+      await client.beginSelectiveRestore("Key Name", backupURI, blobSasToken, {
```

Comment:
Nit: not a valid key name (can't have spaces). Just to avoid potentially confusing customers, maybe `"key-name"`?